develop/ThirdParty/Grafana/Grafana_plugin.md

```diff
@@ -0,0 +1,131 @@
+
+   - **Linux**: Go to `$WORKING_DIR/conf`. If you find the `defaults.ini` file, you have located your Grafana configuration file. If there is no `defaults.ini` file, go to `/usr/local/etc/grafana` and find `grafana.ini`. In this case, `grafana.ini` is your configuration file.
+
+1. Open the configuration file and set the ID of our plugin to `allow_loading_unsigned_plugins`:
+
+    ```ini
+    [plugins]
+    # Enter a comma-separated list of plugin identifiers to identify plugins to load even if they are unsigned. Plugins with modified signatures are never loaded.
+    allow_loading_unsigned_plugins = skylinecommunications-dataminer-datasource
+    ```
+
+1. Save the configuration file.
+
+1. Download the [latest version of the Data Source plugin](https://github.com/SkylineCommunications/dataminer-grafana-plugin/releases). This will be a ZIP file.
+
+1. Extract the entire ZIP file in the *plugins* folder of Grafana:
+
+   - **Windows**: Go to `C:\Program Files\GrafanaLabs\grafana\data\plugins`
+
+   - **Linux**: Go to `/var/lib/grafana/plugins`
+
+1. Restart the Grafana service.
+
+1. Open Grafana and go to *Configuration > Plugins*.
+
+1. Click on *DataMiner* and select *Create a DataMiner data source*.
+
+1. Configure the connection with your DataMiner Agent:
+
+   - User: Specify the username and password that Grafana should use to authenticate on the DMA. These credentials will only provide access to data inside DataMiner as configured in the DataMiner security.
+
+   - HTTP: Specify the URL to connect to your DataMiner Agent, for example `https://mydma.company.com`.
+
+    > [!CAUTION]
+    > Use a secure HTTPS connection. Without an HTTPS connection, your username and password will be sent over the network/internet unencrypted, which means that they may be visible to others.
+    > For more information on how to configure your DataMiner System to use HTTPS, see [Setting up HTTPS on a DMA](xref:Setting_up_HTTPS_on_a_DMA).
+
+   - Other configuration settings can be ignored.
+
+1. Click *Save & test*. A *Success* message should be displayed.
+
+## Features
+
+### Generic Query Interface (GQI)
+
+A GQI query should be specified in JSON format. To get a correctly configured query, you can make use of the [DataMiner Dashboards app](xref:newR_D):
+
+1. In the Dashboards app, create a dashboard and create a query.
+
+   > [!TIP]
+   >
+   > - See [Creating and configuring dashboards](xref:Creating_and_configuring_dashboards).
+   > - See [Configuring query (GQI) data feeds](xref:Configuring_GQI_feeds)
+
+1. Visualize this query on the dashboard.
+
+1. Press F12 to open the developer tools and go to the *Network* tab.
+
+1. Refresh the dashboard and find the *OpenQuerySession* network call.
+
+1. Under payload, right-click *query:* and select *Copy value*.
+
+1. Open Grafana and paste the copied JSON code into Grafana.
```

Comment:
Where in Grafana should they paste this copied JSON code? Could you provide extra steps on how to get to this point?